script/Arcueid/Day 4/04_00_ARC04_1_1.txt

```diff
@@ -91,42 +96,45 @@ Ah, I understand now.
 [sha:daf7ae109dda6fcc900e745da29044cb40542722]{
 -- Page 75, Offset 5749.
 -- 遠野槙久は、秋葉が中学生になっても自由時間を許さなかったのだ。
-Tohno Makihisa didn't allow Akiha any free time even when she was in middle school.
+Tohno Makihisa didn't allow Akiha any free time even after she entered middle school.
 }
 [sha:6ea2dfbba3e9f9592cc68cd15811f3b6d41e1f17]{
 -- Page 76, Offset 5750.
 -- 「………そっか。でも、親父はもういないんだ。秋葉も無理することないんじゃないか？
-"......I see. But he's gone now. There's no reason for you to push yourself anymore, right?
+"......I see. Even so, he's gone now. There's no reason for you to push yourself anymore, right?
 }
 [sha:192a98c7e3946c45aa955d653c893dda2e468aaa]{
 -- Page 76, Offset 5751.
 -- ただでさえ忙しいのに、往復二時間もかけるなんて馬鹿らしい。その分、寄宿舎で友達と過ごした方が楽しいだろ？」
-It's ridiculous to spend two hours traveling back and forth when you're so busy. It would be more fun to spend that time with your friends at the boarding house, right?"
+You're busy enough as it is, it seems crazy to spend two whole hours just going back and forth from school.
+ Plus, wouldn't it be more fun for you to live with your friends in the dorms?"
 }
 [sha:65f187af135d3b43559089988dca0c6e38baa18d]{
 -- Page 77, Offset 5752.
 -- 「ええ、そうできたら苦労はしません。
-"Yes, if I could do that, I would have no troubles.
+"Yes, if only I could.
 }
 [sha:d100ce90aef2e3ecfc8fbdfecfb06e8e3b246d09]{
 -- Page 77, Offset 5753.
 -- ですが、私には責任がありますから。昨日のような事があるかぎり、当分は屋敷から学校に通います」
-But I have responsibilities. As long as incidents like yesterday happen, I will be going to school from the mansion for the time being."
+Unfortunately, I have responsibilities I must attend to.
+ So long as incidents such as yesterday's continue to occur,
+ I will continue to commute to school from the mansion."
 }
 [sha:16dc159f0a510ca7fced85c9905f3ebf52097add]{
 -- Page 78, Offset 5754.
 -- 「昨日のようなこと……？」
-"Incidents like yesterday's...?"
+"Incidents such as yesterday's...?"
 }
 [sha:7a8113e73961aa31e7728a4f5d3ef7b0530c7866]{
 -- Page 78, Offset 5755.
 -- 俺がいない間、遠野家の財産関係で問題でもあったのだろうか？
-Was there a problem with the Tohno family's property while I was gone?
+Did some problem arise with regard to the Tohno family fortune while I was out?
```

Comment:
maybe away instead of out?

Reply:
"away" implies a longer absence imo. I think "out" or "gone" is good